sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -21,6 +21,7 @@ public class PersonalizerClient
         private readonly bool _isLocalInference;
         private string stringEndpoint;
         private string apiKey;
+        private float _SubsampleRate = 1.0f;
```

Comment:
[case] subsampleRate

There is an inconsistency between naming conventions. Please do not use the '_' in variable names.